external-crates/move/move-compiler/transactional-tests/tests/evaluation_order/lazy_assert.move

```diff
@@ -13,11 +13,3 @@ script {
         assert!(false, 1 / 0);
     }
 }
-
-//# run
-script {
-    fun main() {
```

Comment:
(This is my first comment on the PR so maybe I will understand later, but leaving it here for now)
Why did we remove this one?

Reply:
I'm completely unclear what changed, but this is now getting flagged as a diagnostic error instead of a runtime error as it previously did. Note that I didn't touch that code at all, so I'm not sure what caused it.